settings.lua

```diff
@@ -6,13 +6,13 @@ local function setting(tp, name, default)
 	local full_name = "areas."..name
 	local value
 	if tp == "boolean" then
-		value = minetest.setting_getbool(full_name)
+		value = minetest.settings:get_bool(full_name)
 	elseif tp == "string" then
-		value = minetest.setting_get(full_name)
+		value = minetest.settings:get(full_name)
 	elseif tp == "position" then
 		value = minetest.setting_get_pos(full_name)
 	elseif tp == "number" then
-		value = tonumber(minetest.setting_get(full_name))
```

Comment:
Might be worth localizing `minetest.settings` in this file since it's used a lot.